src/JBrowse/Util.js

```diff
@@ -220,6 +220,7 @@ Util = {
     },
 
     resolveUrl: function(baseUrl, relativeUrl) {
+        if(relativeUrl[0] == '/' && this.isElectron()) return relativeUrl;
```

Comment:
Is the electron code feeding absolute filesystem paths to `resolveUrl`? I suspect that should be fixed in a different way.

Reply:
that might be true.n I think there might be some places where it is raw file urls but others where it is file:/// urls (e.g. in the trackList.json from a saved session it is file:/// urls)

i'll look to see if the raw file urls can be avoided

Reply:
I started looking into this but there is a weird thing where if there is a Util.resolveUrl('/', 'file:///some/path') then it would convert it to 'file://some/path' (two slashes instead of three)

Not sure why that is. If I load require('url') in nodejs and do it in a simple script it stays with 3 slashes

Reply:
Meh don't worry about it then, not important enough to block this.